manim/_config/main_utils.py

```diff
@@ -445,6 +449,11 @@ def _parse_args_no_subcmd(args: list) -> argparse.Namespace:
         choices=constants.VERBOSITY_CHOICES,
     )
 
+    parser.add_argument(
+        "--version",
+        help="Print the current version of Manim you are using"
+    )
+
```

Comment:
This shouldn't be necessary since the `--version` flag is being handled manually in `parse_args`. Unless it's here for the purpose of documenting it with `manim -h`.

Reply:
It is for `manim --help`, yes.

Reply:
I believe this is the case[ as discussed on Discord](https://discord.com/channels/581738731934056449/712140817200054294/804062414484144219). I was going to recommend adding a -v shorthand, but then I saw that that's already used for verbosity.